bin/peggy-cli.js

```diff
@@ -173,14 +173,22 @@ class PeggyCLI extends Command {
         "-m, --source-map [mapfile]",
         "Generate a source map. If name is not specified, the source map will be named \"<input_file>.map\" if input is a file and \"source.map\" if input is a standard input. If the special filename `inline` is given, the sourcemap will be embedded in the output file as a data URI.  If the filename is prefixed with `hidden:`, no mapping URL will be included so that the mapping can be specified with an HTTP SourceMap: header.  This option conflicts with the `-t/--test` and `-T/--test-file` options unless `-o/--output` is also specified"
       )
+      .addOption(
+        new Option(
+          "-a, --ast",
+          "Output a grammar AST instead of a parser code"
+        )
+          .default(false)
+          .conflicts(["test", "testFile", "sourceMap"])
+      )
       .option(
         "-S, --start-rule <rule>",
         "When testing, use the given rule as the start rule.  If this rule is not in the allowed start rules, it will be added."
       )
-      .addOption(new Option(
+      .option(
         "-t, --test <text>",
         "Test the parser with the given text, outputting the result of running the parser instead of the parser itself. If the input to be tested is not parsed, the CLI will exit with code 2"
-      ).conflicts("test-file"))
+      )
```

Comment:
I removed this because it not work -- `.conflicts()` [expects a camelCase name](https://github.com/tj/commander.js/blob/dccc90676aa960563ae0d0f3414a64a231286a69/examples/options-conflicts.js#L6)
This works as desired only because `--test-file` also declares `.conflicts("test")`